packages/sdk/src/types.ts

```
@@ -321,6 +321,42 @@ export interface SDK<T extends SchemaType> {
+     * @param {string[]} contract_addresses - Array of contract addresses to filter (empty for all)
+     * @param {string[]} account_addresses - Array of account addresses to filter (empty for all)
+     * @param {Funtion} callback - JavaScript function to call on updates
+     *
+     * # Returns
+     * Result containing subscription handle or error
+     * @returns torii.Subscription
+     */
+    onTokenBalanceUpdated: (
+        contract_addresses: string[],
+        account_addresses: string[],
+        callback: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)